projects/dotcms-field-elements/src/components/dot-checkbox/dot-checkbox.e2e.ts

```diff
@@ -1,11 +1,95 @@
-import { newE2EPage } from '@stencil/core/testing';
+import { newE2EPage, E2EElement, E2EPage } from '@stencil/core/testing';
+import { EventSpy } from '@stencil/core/dist/declarations';
 
-xdescribe('dot-checkbox', () => {
-  it('renders', async () => {
-    const page = await newE2EPage();
+describe('dot-checkbox', () => {
 
-    await page.setContent('<dot-checkbox></dot-checkbox>');
-    const element = await page.find('dot-checkbox');
-    expect(element).toHaveClass('hydrated');
-  });
+    let page: E2EPage;
+    let element: E2EElement;
+    let spyStatusChangeEvent: EventSpy;
+    let spyValueChangeEvent: EventSpy;
+
+    beforeEach(async () => {
+        page = await newE2EPage();
+
+        await page.setContent(`
+        <dot-checkbox
+            name="testName"
+            label="testLabel"
+            hint="testHint"
+            options="valueA|1,valueB|2,valueC|3"
+            value="1"
+            requiredmessage="testErrorMsg"
+            required="true"
+            >
+        </dot-checkbox>`);
+        element = await page.find('dot-checkbox');
+    });
+
+    it('renders', async () => {
+        // tslint:disable-next-line:max-line-length
+        const expectedMarkup = `<dot-checkbox name=\"testName\" label=\"testLabel\" hint=\"testHint\" options=\"valueA|1,valueB|2,valueC|3\" value=\"1\" requiredmessage=\"testErrorMsg\" required=\"true\" class=\"dot-valid dot-pristine dot-untouched hydrated\"><label for=\"testName\">testLabel</label><div><input type=\"checkbox\" id=\"1\" value=\"1\"><label for=\"1\">valueA</label></div><div><input type=\"checkbox\" id=\"1\" value=\"2\"><label for=\"2\">valueB</label></div><div><input type=\"checkbox\" id=\"1\" value=\"3\"><label for=\"3\">valueC</label></div><span class=\"dot-field__hint\">testHint</span></dot-checkbox>`;
+        const hint = await element.find('.dot-field__hint');
+        expect(element.outerHTML).toBe(expectedMarkup);
+        expect(hint).toBeTruthy();
+    });
+
+    it('it should not render hint', async () => {
+        element.setProperty('hint', '');
+        await page.waitForChanges();
+        const hint = await element.find('.dot-field__hint');
+        expect(hint).toBeNull();
+    });
+
+    it('it should have required as false', async () => {
+        element.setProperty('required', 'false');
+        await page.waitForChanges();
+        const required = await element.getProperty('required');
+        expect(required).toBeFalsy();
```

Comment:
here we doing in the markup `required="fase"` or required doesn't show at all?

Reply:
required="false"

Reply:
https://github.com/dotCMS/core-web/pull/999#discussion_r278775027